yarn.lock

```diff
@@ -7,6 +7,13 @@
   resolved "https://registry.yarnpkg.com/7zip-bin/-/7zip-bin-5.1.1.tgz#9274ec7460652f9c632c59addf24efb1684ef876"
   integrity sha512-sAP4LldeWNz0lNzmTird3uWfFDWWTeg6V/MsmyyLR9X1idwKBWIgt/ZvinqQldJm3LecKEs1emkbquO6PCiLVQ==
 
+"@ampproject/remapping@^2.1.0":
```

Comment:
just checking, is this big set of changes to yarn.lock expected?

Reply:
It is because currently `fetch-mock` is being brought in. But as @jansav mentioned that is less than ideal and it would be better if the injectables were more focused. So this should go away.